### ISSUE TITLE
Proposal: Decrease detection of engine_disconnect

### ISSUE THREAD

**Issue by aanm** [docker/swarm#1030]
Is it possible to decrease the detection time of an engine?
I was testing the amount of time swarm would take until notices that a particular engine was down.
```
2015-07-06T10:43:33.000000000+01:00 006689afe9b2cebb3a02cdc93808eeb78def2f0e15060c261ceed59ed9fa345f: (from hello-world:latest node:localhost.localdomain) die
2015-07-06T11:04:57.000000000+01:00 (from swarm node:node1) engine_disconnect
```
I shut down the node 1 engine a couple seconds after running the hello-world image so I assume it has a 20 minute time out to detect if a particular engine was down. Can't we put around 30 seconds or less?

**Comment by aluzzardi** [comment]
We could and should add a flag to do that.

/cc @vieux @abronan

**Comment by aluzzardi** [comment]
Implemented (finally!) in #1363
